src/sampler.jl

```diff
@@ -94,7 +94,7 @@ end
 Progress meter update with all trajectory stats, iteration number and metric shown.
 """
 function pm_next!(pm, stat::NamedTuple)
-    ProgressMeter.next!(pm; showvalues = [tuple(s...) for s in pairs(stat)])
+    ProgressMeter.next!(pm; showvalues = map(tuple, values(stat), keys(stat)))
```

Comment:
It looks good; why does the current code allocate more?

Reply:
It creates an array instead of a tuple.